tests/unit/test_utils.py

```diff
@@ -261,3 +273,15 @@ def test_get_journal_and_section_invalid():
 
     assert journal_title == ''
     assert section == ''
+
+
+def test_strict_kwargs_pass(dummy_strict_kwargs_cls):
+    """Test the `strict_kwargs` decorator allowing the kwargs."""
+    dummy = dummy_strict_kwargs_cls(a=1, b=2, d=None, _x=4, settings={'DUMMY': True})
+    assert callable(dummy.__init__)
```

Comment:
Here you should test not that it's callable, but that the params were correctly passed to it, probably changing the class to save the values passed and then asserting against it.